Resources/Prototypes/DeltaV/Entities/Structures/Walls/railing.yml

```diff
@@ -37,3 +37,127 @@
     sprite: DeltaV/Structures/Walls/railing.rsi
   - type: Icon
     sprite: DeltaV/Structures/Walls/railing.rsi
+
+- type: entity
+  parent: Railing
+  id: RailingDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_grey
+
+- type: entity
+  parent: RailingCorner
+  id: RailingCornerDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_grey
+
+- type: entity
+  parent: RailingCornerSmall
+  id: RailingCornerSmallDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_grey
+
+- type: entity
+  parent: RailingRound
+  id: RailingRoundDeltaVGrey
+  suffix: Modern, Grey
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: round_grey
+
+- type: entity
+  id: BaseRailingWood
+  suffix: Modern, Wood
+  components:
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+        - !type:DoActsBehavior
+          acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 20
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: WoodDestroy
+          params:
+            volume: -6
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWoodPlank1:
+            min: 0
+            max: 1
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+
+- type: entity
+  parent: [BaseRailingWood, Railing]
+  id: RailingDeltaVWood
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_wood
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: side_wood
+  
+
+- type: entity
+  parent: [BaseRailingWood, RailingCorner]
+  id: RailingCornerDeltaVWood
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_wood
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_wood
+
+- type: entity
+  parent: [BaseRailingWood, RailingCornerSmall]
+  id: RailingCornerSmallDeltaVWood
+  suffix: Modern
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_wood
+  - type: Icon
+    sprite: DeltaV/Structures/Walls/railing.rsi
+    state: corner_small_wood
+
+- type: entity
+  parent: [BaseRailingWood, RailingRound]
+  id: RailingRoundDeltaVWood
+  suffix: Modern
```

Comment:
remove suffix for all these, BaseRailingWood has it

Reply:
oops i did it for two but missed the other two👍